kategory/src/test/kotlin/kategory/data/OptionTTest.kt

```diff
@@ -29,6 +30,11 @@ class OptionTTest : UnitSpec() {
                 Eq.any(),
                 Eq.any()))
 
+        testLaws(FunctorFilterLaws.laws(
+                OptionT.applicative(Option),
+                OptionT.functorFilter(),
+                Eq.any()))
```

Comment:
I believe it may need an instance of Eq<OptionTPartial<Option, Int>>, as we've seen similar issues with other tests before.